mmv1/products/compute/Disk.yaml

```diff
@@ -431,6 +432,10 @@ properties:
       Indicates how many IOPS must be provisioned for the disk.
     required: false
     default_from_api: true
+    description: |
```

Comment:
This duplicates the above description, and will likely cause a compilation error